### PR TITLE
Switch the format a bit, and add some files.

### PR DESCRIPTION
**Pull request by bwinton**
How does this look to you, Amin?

### CODE REVIEW

File: photon-icons/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "scripts": {
+    "build": "node build.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+  },
+  "keywords": [
+    "firefox",
+    "ant",
```
Comment (aminalhazwani) [inline photon-icons/package.json]: This is the only thing that I find odd, probably we could mention tokens or similar instead

**Review by aminalhazwani — APPROVED** [review]

File: photon-icons/package.json (hunk 1 above)
Comment (bwinton) [reply photon-icons/package.json]: How about:
``` javascript
  "keywords": [
    "firefox",
    "icons",
    "design",
    "tokens",
    "json"
  ],
```

File: photon-icons/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+  },
+  "keywords": [
+    "firefox",
+    "icons",
+    "design",
+    "tokens",
+    "json"
+  ],
+  "author": "Blake Winton <bwinton@latte.ca>",
```
Comment (jens1o) [inline photon-icons/package.json:19]: Out of curiosity, why there isn't an `@mozilla.org` email?

**Review by jens1o — COMMENTED** [review]
Question incoming :D

File: photon-icons/package.json (hunk 2 above)
Comment (bwinton) [reply photon-icons/package.json:19]: No reason. 🙂

**Pull request merged**